### DISCUSSION TITLE
LAB 02.04 - CARD TRICK task 2. El autocorrector me da una respuesta negativa trivial.

### DISCUSSION THREAD

**Discussion by Danico3** [rramosp/ai4eng.v1#232] [02 - PYTHON]
Hola profe y monitores.
Hace rato estoy intentando que el auto corrector me acepte mi solución sin tener éxito. No sé que más pueda intentar. Me pueden dar luces, por favor, de cómo más puedo abordar este código.
Gracias

Autocorrector
![image](https://user-images.githubusercontent.com/124913374/222017166-02714fac-451a-4a31-9aa5-fdf952ef75dd.png)

Función

    def collect_heaps(h, n):
        h = np.array(h)
        look = np.where(h == n)[0]
        heaps_mix = np.arange(0,h.shape[0])
        np.random.shuffle(heaps_mix)
        while heaps_mix[h.shape[0]//2] != look:
          np.random.shuffle(heaps_mix)
        
        r = [h[i] for i in heaps_mix] 
        #r.flatten()
    
        return r[h.shape[0]//2].tolist()

**Comment by Felipe-RA** [comment]
Saludos

El problema se encuentra en lo que estás devolviendo, 

Tú estas devolviendo sólo el heap con la carta elegida, pero debes devolver todo el deck (organizado con el heap que tiene la carta elegida en el medio)

**Comment by ljaimesm74** [comment]
Hola, 
como se hace: "Tú estas devolviendo sólo el heap con la carta elegida, pero debes devolver todo el deck (organizado con el heap que tiene la carta elegida en el medio)"
